share/spack/gitlab/cloud_pipelines/configs/darwin/aarch64/compilers.yaml

```diff
@@ -0,0 +1,27 @@
+compilers:
+- compiler:
+    spec: apple-clang@14.0.0
+    paths:
+      cc: /usr/bin/clang
+      cxx: /usr/bin/clang++
+      f77: /opt/homebrew/bin/gfortran
+      fc: /opt/homebrew/bin/gfortran
+    flags: {}
+    operating_system: ventura
+    target: aarch64
+    modules: []
+    environment: {}
+    extra_rpaths: []
+- compiler:
+    spec: gcc@12.2.0
+    paths:
+      cc: /opt/homebrew/bin/gcc-12
+      cxx: /opt/homebrew/bin/g++-12
+      f77: /opt/homebrew/bin/gfortran-12
+      fc: /opt/homebrew/bin/gfortran-12
+    flags: {}
+    operating_system: ventura
+    target: aarch64
+    modules: []
+    environment: {}
+    extra_rpaths: []
```

Comment:
Could you add EOL chars before the EOF char in all files? That's what this red circle means.

Reply:
Yes, done!

Reply:
Seems to still be present in a lot of files

Reply:
You are right. My reading comprehension is lacking. It should now in fact be removed from "ALL" files :)